torchci/rockset/commons/__sql/flaky_tests.sql

```diff
@@ -0,0 +1,27 @@
+SELECT
```

Comment:
So, this query gets the aggregate # of reds/greens across all different test types and configurations in the time window?

I would have thought that we would group on a per-job basis, so that we can differentiate between jobs (for example, a test might be flaky on windows but not on linux). But I guess this is fine for now.

Reply:
This query returns the same stuff as the OG flaky test query which gives job names and such so we can still make the same classifications (we currently figure out platform differences in the code itself). I opted for this way as it'd be easiest to port over and verify. We can definitely improve it later on.